docs/404.md

```diff
@@ -0,0 +1,8 @@
+<frontmatter>
+    title: Page not found
+    layout: 404.md
+</frontmatter>
+
+
```

Comment:
May I check if this blank line is necessary?

Reply:
Not necessary

Reply:
If so could we remove it? What do you think?

Reply:
I will update it. 😢